plugins/tags/tag_visitor.cpp

```diff
@@ -81,13 +84,30 @@ namespace golos { namespace plugins { namespace tags {
         db_.remove(tag);
     }
 
+    comment_date operation_visitor::get_comment_last_update(const comment_object& comment) const {
+        if (db_.has_index<comment_last_update_index>()) {
+            const auto& clu_idx = db_.get_index<comment_last_update_index>().indices().get<golos::plugins::social_network::by_comment>();
+            auto clu_itr = clu_idx.find(comment.id);
+            if (clu_itr != clu_idx.end()) {
+                comment_date result;
+                result.active = clu_itr->active;
+                result.last_update = clu_itr->last_update;
+                return result;
+            }
+        }
+        return comment_date();
```

Comment:
For compiler better if you declare `comment_date result;`, fill its fields if it exists, and return.
In this case it will copy-elision.